### DISCUSSION TITLE
What is your hobby?

### DISCUSSION THREAD

**Discussion by mmm-nana** [NAIST-SE/AttractingContributors#473] [General]
Please feel free to tell me your hobby.
Thank you.
my rep: https://github.com/mmm-nana/AttractingContributors-hobby

**Comment by aru1702** [comment]
I like listening music, play them a little, and reading books, like manga or artbook
How about you?

See my repo: https://github.com/aru1702/AttractingContributors-TouhouFumo
Hop into my discussion: https://github.com/NAIST-SE/AttractingContributors/discussions/407
